libkbfs/init.go

```diff
@@ -616,11 +616,13 @@ func doInit(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn,
 		log.Warning("Could not enable disk limiter: %+v", err)
 		return nil, err
 	}
+	goctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
Ah I see, the "go" referred to "golang", not "goroutine".  Cool.  `ctx10s` still seems better to me, thanks.